docs/semgrep-pro-vs-oss.md

```diff
@@ -0,0 +1,203 @@
+| Send scan results to GitLab SAST and GitHub Advanced Security                                                   | --          | ✔️                |
+| [Findings dashboard](/semgrep-cloud-platform/dashboard/)                                                        | --          | ✔️                |
+| Findings retention                                                                                              | --          | 5 years           |
+
+## 🧰 Scan customization features
+
+The following table displays customization features and tools that enhance Semgrep's core scanning capabilities. These features can increase true-positive rate and provide deeper insights into remediation.
+
+| Feature                                                      | Semgrep OSS                                     | Semgrep Pro                            |
+| ------------------------------------------------------------ | ----------------------------------------------- | -------------------------------------------- |
+| Write your own rules                                         | ✔️                                              | ✔️                                           |
+| [Community-contributed rule registry](https://semgrep.dev/r) | ✔️                                              | ✔️                                           |
+| Rule-writing environment                                     | ✔️ [Playground](https://semgrep.dev/playground) | ✔️ Playground and Editor for logged-in users |
+| Private rules\*                                              | n/a                                             | ✔️                                           |
+| Proprietary rule registry                                    | --                                              | ✔️                                           |
+| [Policy-based workflows†](/semgrep-code/policies/)           | --                                              | ✔️                                           |
+
+\*Private rules refer to rules that are guaranteed a private access scope in the cloud. This scope of access does not apply to Semgrep OSS, as Semgrep OSS is purely CLI-based.<br />
+† Policy-based workflows provide security teams a means to block merges, leave PR/MR comments, or silently monitor for potential issues based on the presence of a finding.
+
+### 🤖 Developer experience
+
+The following table lists tools to enable developers to resolve their own code.
+
+| Feature                   | Semgrep OSS | Semgrep Pro |
+| ------------------------- | ----------- | ----------------- |
+| VS Code extension         | ✔️          | ✔️                |
+| IntelliJ extension        | ✔️          | ✔️                |
+| `pre-commit`‡             | ✔️          | ✔️                |
+| Autofix                   | ✔️          | ✔️                |
+| Autofix in PR/MR comments | --          | ✔️                |
+| Autofix AI                | --          | ✔️                |
```

Comment:
Is this the term we normally use? Feels weird, I would call it "Assistant autofix" or "AI autofix"

Reply:
Good callout. We usually call it GPT-assisted autofix.